### ISSUE TITLE
Feature Request: Add create timeouts argument to ibm_network_vlan

### ISSUE THREAD

**Issue by bavibm** [IBM-Cloud/terraform-provider-ibm#994]
The `ibm_network_vlan` resource has no `create` argument in the `timeouts` block. Is this intentional? If it is intentional, I'd like to request to add this feature.

Our particular issue is very specific:

Our SoftLayer backend is rather slow to update its backend order information store, so whenever we create a new VLAN resource it will successfully place an order and the plugin code will receive an order number but timeout before it can retrieve the order details. I've monitored the debug output and found that the issue is likely around line 175 in the plugin code [here](https://github.com/IBM-Cloud/terraform-provider-ibm/blob/master/ibm/resource_ibm_network_vlan.go). Our endpoint keeps sending an empty response to this query until it's database is updated - which sometimes takes longer than the default retry time period (21 retries in 10-second intervals I believe). We'd like to be able to tell Terraform to wait until our order details are received before continuing forward rather than having to restart the program because it timed out and terminated prematurely.

### Terraform Version
```terminal
Terraform v0.12.18
+ provider.ibm v1.0.0
```

But this was also an issue in previous Terraform and Plugin versions

### Affected Resource(s)
- `ibm_network_vlan`

### Terraform Configuration Files
```hcl
resource "ibm_network_vlan" "private" {
  count      = var.new_vlan ? 1 : 0
  name       = "prv-${var.random_id}"
  datacenter = var.datacenter
  type       = "PRIVATE"
  timeouts {
    delete = "${var.wait_time}m"
    create = "${var.wait_time}m"
  }
}
```

Additional variable and provider blocks omitted...

### Debug Output
Running terraform validate produces the output
[https://gist.github.com/bavibm/238a5508cc5397e96c15a30884b44099](https://gist.github.com/bavibm/238a5508cc5397e96c15a30884b44099)

### Expected Behavior
Terraform should validate the configuration successfully

### Actual Behavior
Terraform validate failed with the above error message

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform init`
2. `terraform validate` or `terraform apply`

### Important Factoids
None

### References
None

**Comment by hkantare** [comment]
Fixed in latest releases v1.1.0 and v0.23.0
https://github.com/IBM-Cloud/terraform-provider-ibm/releases

**Comment by bavibm** [comment]
Awesome, thanks! I will be reporting back if there are any subsequent issues, but this can likely be closed out now.
